src/index.ts

```diff
@@ -119,11 +142,25 @@ export class IDX {
   _toDefinitionId(name: string): DocID {
     const id = this._definitions[name] ?? this._definitions[`idx:${name}`]
     if (id == null) {
+      if (name.startsWith('ceramic://')) {
+        return name
+      }
```

Comment:
:+1: Once we update to proper DocIDs we'll have a tool that detects if a string is a docID (won't have the prefix anymore).